charts/nirmata/values.yaml

```diff
@@ -393,9 +400,12 @@ config:
 
 
   # -- Defines annotations to set on webhook configurations.
-  webhookAnnotations: {}
-  # Example to disable admission enforcer on AKS:
-  # 'admissions.enforcer/disabled': 'true'
+  webhookAnnotations:
```

Comment:
same as for the above security context. Add this to the kyverno.helm parameters in cr-kyverno.yaml in enterprise-kyverno-operator chart.